src/microsoft/office/license/user/USER.html

```diff
@@ -10,103 +10,38 @@
-                                    <span class="oui-icon oui-icon-gear_line oui-icon_small" aria-hidden="true"></span>
-                                </button>
-                            </td>
-                        </tr>
-                    </tbody>
-                </table>
-            </div>
-
-            <div class="clearfix"
-                 data-ng-if="usersCtrl.users">
-                <div data-pagination-front
-                     data-items="usersCtrl.users"
-                     data-paginated-items="usersCtrl.usersDetails"
-                     data-current-page="currentPageUsers"
-                     data-nb-pages="nbPages"
-                     data-items-per-page="itemsPerPage"
-                     data-transform-item="usersCtrl.transformItem(item)"
-                     data-on-transform-item-done="usersCtrl.onTransformItemDone(items)"
-                     data-page-placeholder="{{tr('pagination_page', [currentPageDynHosts, nbPages])}}"
-                     data-item-per-page-placeholder="{{tr('pagination_display')}}"
-                     data-ng-show="!usersCtrl.loaders.users && usersCtrl.usersDetails.length > 0">
-                </div>
-            </div>
+            <oui-datagrid data-rows="usersCtrl.users"
+                          data-row-loader="usersCtrl.transformItem($row.user)">
+                <oui-column data-title="tr('microsoft_office_license_detail_user_firstname')" data-property="firstName"></oui-column>
+                <oui-column data-title="tr('microsoft_office_license_detail_user_lastname')" data-property="lastName"></oui-column>
+                <oui-column data-title="tr('microsoft_office_license_detail_user_activationEmail')" data-property="activationEmail"></oui-column>
+                <oui-column data-title="tr('microsoft_office_license_detail_user_licenses')" data-property="licences">
+                    <span data-ng-repeat="licence in $row.licences track by $index" data-ng-bind="licence"></span>
+                </oui-column>
+                <oui-column data-title="tr('microsoft_office_license_detail_user_status')" data-property="">
+                    <span class="label"
+                          data-ng-if="!$row.isVirtual"
+                          data-ng-bind="tr('microsoft_office_license_user_status_' + $row.status)"
+                          data-ng-class="{
+                            'label-success': $row.status === 'ok',
+                            'label-danger': $row.status === 'deleting',
+                            'label-info': $row.status === 'creating'}"></span>
+                    <span class="label label-info"
+                          data-ng-if="$row.isVirtual"
+                          data-i18n-static="microsoft_office_license_detail_user_not_configure">
+                    </span>
+                </oui-column>
+                <oui-action-menu data-align="end" data-compact data-ng-if="!usersCtrl.loaders.userPoll[$row.activationEmail]">
```

Comment:
i'm not sure about the behaviour that the action menu will have based on a if with $row information.

Reply:
saw with you. this works. Just replacing ng-if by ng-hide to avoid reloading data.